dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -1294,18 +1295,6 @@ protected List<Node> convertCallArguments(
           convertedNodes.add(methodInvocationConvert(actualVal, formals.get(i)));
         }
 
-        // NOTE: When the last parameter is a type variable vararg and the compiler
-        // cannot find a specific type use to substitute for it, the compiler will
-        // create an unbounded component type instead. For example,
-        // for the following method declaration:
-        // <T> void foo(T... ts) {}
-        // consider this method invocation:
-        // foo();
-        //
-        // At the call site, the compiler doesn't have enough information about the
```

Comment:
I didin't see this when debugging the example, so I deleted the comment.  (In the example, T is substituted with `Object`.)